### DISCUSSION TITLE
FHIR search queries within the prefetch template

### DISCUSSION THREAD

**Discussion by mseckykoebel** [flexpa/sero#55] [Q&A] [answered]
I have a sample patient EHR bundle that I downloaded from [Synthea](https://synthea.mitre.org/downloads) with lots of different resources (Encounters, Conditions, Observations, etc.). I am trying to make a service that collects all the observations inside of this bundle - more specifically, one that gathers all of the times a blood pressure measurement was taken. My search query looks like this:

```
  {
    ...
    prefetch: {
      bloodPressureObservations: "Observation?code=http://loinc|55284-4",
    },
    ...
  },
```
I am using the [logica sandbox](https://sandbox.logicahealth.org/) as my test client. Inside of the bundle, there are 10 total observations where blood pressure was taken for the patient (with corresponding loinc code). However, this query does not return anything. Below is the response body corresponding with this request:

```
{"resourceType":"Bundle","id":"9d139a4e-f00d-444e-b3ad-adbe51f56281","meta":{"lastUpdated":"2021-07-27T21:51:47.516+00:00"},"type":"searchset","total":0,"link":{"relation":"self","url":"https://api.logicahealth.org/APIGuideTesting/data/Observation?code=http%3A%2F%2Floinc%7C55284-4"}}
```

I am expecting this response to contain the 10 observation resources. Any ideas how this search could be modified to accomplish this?

**Comment by jdjkelly** [comment, marked as answer]
We talked about this in a paired session.

The bug ended up being `http://loinc|55284-4"` vs `http://loinc.org|55284-4"`
